### ISSUE TITLE
API docs are lacking

### ISSUE THREAD

**Issue by wcooley** [kevin1024/vcrpy#420]
The "API" section of the documentation is a bit thin. For example, [`vcr.request.Request`](https://vcrpy.readthedocs.io/en/v2.0.1/api.html#module-vcr.request) only includes the `__init__` method, not any of the properties or methods.

I presume this is because there are no docstrings in the missing methods and the "undoc-members" is not enabled in the [autodoc default options](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html?highlight=autodoc#confval-autodoc_default_options). Knowing that the method exists and seeing the signature would be better than nothing, even if there aren't actual docstrings.

I think all that would need to be done is to add to `conf.py`:
```python
autodoc_default_options = {
    'members': None,
    'undoc-members': None,
}
```

**Comment by neozenith** [comment]
Happy to  accept a pull request if you have the time to assist with this

**Comment by neozenith** [comment]
I don't have the bandwidth to chase this one up. Will accept PRs though.

**Comment by neozenith** [comment]
Actually, scratch that. I was added as a maintainer of the RTD project now and it's bugging me that this isn't sorted. I'll chase this up.

**Comment by neozenith** [comment]
This is getting addressed in #499
